swagger.json

```diff
@@ -365,6 +365,30 @@
           }
         }
       }
+    },
+    "/version": {
+      "get": {
+        "description": "Return blackbeard, kubernetes and kubectl version",
+        "summary": "Return blackbeard, kubernetes and kubectl version",
+        "operationId": "get-version",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Version",
+            "schema": {
```

Comment:
This swagger spec is invalid. You should keep an eye on [https://github.com/go-swagger/go-swagger](https://github.com/go-swagger/go-swagger)

Then : 

```sh
swagger validate swagger.json
```

Here is the installation guide : [https://goswagger.io/install.html](https://goswagger.io/install.html)